drivers/display/display_st7735r.c

```diff
@@ -147,8 +149,19 @@ static int st7735r_set_mem_area(const struct device *dev,
 		return ret;
 	}
 
-	uint16_t ram_x = x + data->x_offset;
-	uint16_t ram_y = y + data->y_offset;
+	uint16_t ram_x;
+	uint16_t ram_y;
+
+	if (
```

Comment:
Very much a nit, but can you make formatting consistent between this and the other `if` added below? I'd prefer to have the condition statement start on the same line as the `if`, like the other addition

Reply:
Thanks @danieldegrasse - updated.